spec/library_spec.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/library.rb'
+require 'date'
+
+describe Library do
+   books = [{ title: 'Alfons leker', author: 'A. Andersson', status: :okay, person: :non, duedate: ''},{ title: 'Bible', author: 'TGAOTU', status: :lended , person: :MEEE, duedate: "2016-03-01"} ]
```

Comment:
Metrics/LineLength: Line is too long. [196/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)